### DISCUSSION TITLE
ZWO 715MC connected via PoE to Pi5---Indie or Libcamera (or other).

### DISCUSSION THREAD

**Discussion by stevehookem** [aaronwmorris/indi-allsky#1317] [General]
I have tried both. Indie just tries to run a script all the time and says indieserver not found. The Libcamera option gives many selections but my camera isn't there. What have I done? Should I erase everything and start over? If so, how? lol

**Comment by stevehookem** [comment]
this is where I am at now.
![allskyindie](https://github.com/aaronwmorris/indi-allsky/assets/167378571/a0c378f7-5ca7-4912-bf62-3defc0622ec6)

**Comment by aaronwmorris** [comment]
You would select `1` for indi and when you define the driver for the indiserver, you should select `indi_asi_ccd` or `indi_asi_single_ccd`.

If you have run the setup previously, you will need to restart the indiserver when finished...

```
systemctl --user restart indiserver
```

If you are still having problems, post the output of the Support Info view or the output of `./misc/support_info.sh` command.

**Reply by stevehookem** [reply]
Thanks! I was able to select the driver this time. Then it asks about Python and some others that I selected no. At the end, it is an error
![indiallsky2](https://github.com/aaronwmorris/indi-allsky/assets/167378571/c29148c8-876c-483c-8de8-9b7aa13c17b6)

**Reply by aaronwmorris** [reply]
What OS are you running?

**Reply by stevehookem** [reply]
Windows 11 on PC
Bookworm on RPi 5 with PoE Hat

**Reply by aaronwmorris** [reply]
Can you show the output of this?

```
sudo apache2ctl configtest
```

**Reply by stevehookem** [reply]
stevehookem@allsky:~/indi-allsky $ sudo apache2ctl configtest
AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 127.0.1.1. Set the 'ServerName' directive globally to suppress this message
Syntax OK
stevehookem@allsky:~/indi-allsky $

**Reply by stevehookem** [reply]
![indiallsky3](https://github.com/aaronwmorris/indi-allsky/assets/167378571/4b0f03f8-0afb-4ebc-802f-dd2734f53d3e)

**Reply by stevehookem** [reply]
I appreciate the help! I don't know what I am doing. I did have the other Allsky running. It can be done!

**Reply by aaronwmorris** [reply]
No worries.  Can you show the following output?

```
sudo ss -antp|grep 80
```

**Reply by stevehookem** [reply]
![Uploading indiallsky4.png…]()

**Reply by stevehookem** [reply]
![indiallsky4](https://github.com/aaronwmorris/indi-allsky/assets/167378571/24b0d430-db6d-45fd-87d6-aa12ab50cada)

**Comment by stevehookem** [comment]
stevehookem@allsky:~/indi-allsky $ sudo ss -antp|grep 80
LISTEN    0      1024                            0.0.0.0:80                       0.0.0.0:*     users:(("lighttpd",pid=847,fd=5))
LISTEN    0      128                             0.0.0.0:22                       0.0.0.0:*     users:(("sshd",pid=807,fd=3))
TIME-WAIT 0      0                        192.168.50.254:80                 192.168.50.95:65115

TIME-WAIT 0      0                        192.168.50.254:80                 192.168.50.95:65089

TIME-WAIT 0      0                        192.168.50.254:80                 192.168.50.95:65099

LISTEN    0      1024                               [::]:80                          [::]:*     users:(("lighttpd",pid=847,fd=4))
LISTEN    0      128                                [::]:22                          [::]:*     users:(("sshd",pid=807,fd=4))
ESTAB     0      0      [fe80::2170:c30e:2a66:277f]%eth0:22   [fe80::3aa2:52bc:27cf:6e60]:62800 users:(("sshd",pid=2816971,fd=4),("sshd",pid=2816965,fd=4))

**Reply by aaronwmorris** [reply]
Ah, you had TJ's allsky running on here.

Let's shutdown lighttpd for the time being, you can always re-enable it later.

```
sudo systemctl stop lighttpd

sudo systemctl disable lighttpd
```

Then re-run `setup.sh`

**Reply by stevehookem** [reply]
OK! It says enjoy! Now what? lol I still see live views from the other ALLSky software

**Reply by aaronwmorris** [reply]
You should be able to start the indi-allsky capture process.

```
systemctl --user start indi-allsky
```

After a few minutes, you should start seeing images on the web interface:

https://allsky.local/indi-allsky/

If you are still having problems, post the output of the Support Info view or the output of `./misc/support_info.sh` command.

**Reply by stevehookem** [reply]
Seems to be working now. Thanks.
Question: in the other AllSky app, I could delete the previous images and timelapse movies by clicking a trashcan to delete. I can't figure out how to delete them. Is there an easy way?

**Reply by stevehookem** [reply]
Also getting a message that "sensorworker" was restarted more than 10 times. What is that?

**Reply by aaronwmorris** [reply]
I am not sure about deleting previous images and timelapses from TJ's allsky.  It has been a while since I tested the project.

The Sensor worker is the process that takes readings from physical sensors connected to your system.  Make sure you have all of your sensors set to "None" on the Sensors/Devices tab if you do not have any of those devices.

**Reply by stevehookem** [reply]
> I am not sure about deleting previous images and timelapses from TJ's allsky. It has been a while since I tested the project.
> 
> The Sensor worker is the process that takes readings from physical sensors connected to your system. Make sure you have all of your sensors set to "None" on the Sensors/Devices tab if you do not have any of those devices.

How do I delete the image and timelapse files on Indi?
Also, I do have a fan on my RPi 5 PoE Hat. How do I make sure it is running 100%? It's hot in Texas!

**Reply by aaronwmorris** [reply]
If you just want to nuke all of the images and timelapses, you can navigate to System -> Utilities and then `Flush Images` and then `Flush Timelapses`.

Do you have a link to the PoE hat?  I will see if the fan is user controllable.

**Reply by stevehookem** [reply]
Thanks, here's the hat: https://a.co/d/8hhQFQ8

**Reply by stevehookem** [reply]
Any luck?

**Reply by stevehookem** [reply]
Another stupid question, sorry! How do I get back in by command line? When I put my user name and IP of the camera, it opens username@allsky.local. I think this is the other AllSky, not indi. When I try username@indi-allsky.local, it doesn't find that. My camera is up and running.

**Reply by aaronwmorris** [reply]
According to the info above, I believe username@allsky.local should be what you should be using.

As for the fan, I think it is controlled via i2c, but I am not sure.  Can you run `i2cdetect -y 1` and paste the output here?  You may also have to install i2c-tools as well.

```
sudo apt-get install i2c-tools

i2cdetect -y 1
```

**Reply by stevehookem** [reply]
OK so going to user@allsky.local gets me to the INDI allsky? When I installed the INDI version, did it remove the old ALLSKY from Thomas Jacquin? I was running that before. Do I still need to run ~/indi-allsky?

Here is the output if I did it correctly. 
![indiallsky5](https://github.com/aaronwmorris/indi-allsky/assets/167378571/187515f3-fd2b-4ef2-860a-116970c9aaa5)

**Reply by stevehookem** [reply]
After running the tools, I have this:
![indiallsky6](https://github.com/aaronwmorris/indi-allsky/assets/167378571/015aa5d5-3b07-48c1-bcab-6cd26bf2cf0f)

**Reply by aaronwmorris** [reply]
You just missed the 1 at the end of the command.

```
i2c-detect -y 1
```

^^^ typo

**Reply by stevehookem** [reply]
command not found. I did have the 1 on the first try. I ran it now with the 1 and it is not found.

**Reply by aaronwmorris** [reply]
That was my fault, I typoed the command

```
i2cdetect -y 1
```

**Reply by stevehookem** [reply]
![indiallsky7](https://github.com/aaronwmorris/indi-allsky/assets/167378571/bd2d39ce-1c7f-4d51-8336-d9d47800e78a)

**Reply by aaronwmorris** [reply]
There are no schematics for the board, but just following the traces in the picture, it looks like the fan is just hardwired to the 5v pin.  Does the fan ever turn off?

**Reply by stevehookem** [reply]
It does run low all the time
